app/src/main/java/de/westnordost/streetcomplete/quests/AbstractQuestForm.kt

```diff
@@ -169,6 +170,13 @@ abstract class AbstractQuestForm :
         updateInfoButtonVisibility()
     }
 
+    protected fun setObjNote(text: CharSequence?) {
+        binding.titleNoteLabel.isGone = text == null
+        binding.titleNoteLabel.text = getString(R.string.note_for_object)
+        binding.noteLabel.isGone = text == null
+        binding.noteLabel.text = text
+        decideObjNoteVisible()
+    }
```

Comment:
This can be simplified. The title is static, it could be specified in the layout XML. Also, no need to set these to gone or not, as the container is already set to gone or not.

```suggestion
    protected fun setObjNote(text: CharSequence?) {
        binding.noteLabel.text = text
        binding.speechbubbleNoteContainer.isGone = text == null
    }
```

Reply:
Resolved with afcb138, but had to use text.isEmpty() instead of text == null because of calling setObj without prior null check.